src/Microsoft.TemplateEngine.Cli/HelpAndUsage/TemplateUsageHelp.cs

```diff
@@ -65,6 +65,10 @@ public static void ShowInvocationExamples(ListOrHelpTemplateListResolutionResult
 
             // show a help example
             Reporter.Output.WriteLine($"    dotnet {commandName} --help");
+
+            // show a help example for template
+            string preferredTemplateName = bestMatchedTemplates.First().Info.ShortName;
```

Comment:
minor: if ```.First()``` return null, are ok with the resulting thrown exception?

Reply:
It looks like `bestMatchedTemplates` is checked for being empty (line 25). And the code already assumes that its elements are non-`null` so `First()` should never return `null` here.